content/en/docs/tasks/administer-cluster/encrypt-data.md

```diff
@@ -45,15 +45,24 @@ decrypt data stored in the etcd.
 ## Understanding the encryption at rest configuration
 
 ```yaml
+---
```

Comment:
This looks a bit odd in the preview.

Reply:
I'll explain my thinking here:
- I'd like us (SIG Docs) to start all YAML that represents an _entire_ document with `---`
- YAML fragments should never start with `---`

The idea is that some readers can immediately tell which are which, and if not then we can one day have ways to automate providing useful hints.

Reply:
starting a yaml with `---` is common practice and preferred by many